python/cugraph/utilities/utils.py

```diff
@@ -173,6 +176,72 @@ def get_traversed_path_list(df, id):
     return answer
 
 
+def get_traversed_cost(df, source_col, dest_col, value_col):
+    """
+    Take the DataFrame result from a BFS or SSSP function call and sums
+    the given weights along the path to the starting vertex.
+
+    Input Parameters
+    ----------
+    df : cudf.DataFrame
+        The dataframe containing the results of a BFS or SSSP call
+    source_col : cudf.DataFrame
```

Comment:
what's the numbering assumption for these Series?

Reply:
They need to match the vertex ids in "predecessor", "vertex" from SSSP/BFS output: https://github.com/rapidsai/cugraph/pull/1376/commits/2c3fc8bdaf9174d71e7842e1f70e571656dc0b7c

Reply:
Sounds good. Just to confirm, they do not need to be continuous or starting at 0 (ie. unnumbered data is accepted) right?

Reply:
Correct, unrenumbered is accepted.